Example in the readme might be missing an "using" statement

### Steps to reproduce

try to setup pomelo as specified by the example in the readme

### The issue

The project doesn't build because ServerVersion is not in scope.

```
Microsoft (R) Build Engine version 16.4.0+e901037fe for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 23.96 ms for /home/abraham/src/OpenPaymentsService/HubDePagos.csproj.
Installers/DbInstaller.cs(19,68): error CS0246: The type or namespace name 'ServerVersion' could not be found (are you missing a using directive or an assembly reference?) [/home/abraham/src/OpenPaymentsService/HubDePagos.csproj]
Startup.cs(54,13): warning MVC1005: Using 'UseMvc' to configure MVC is not supported while using Endpoint Routing. To continue using 'UseMvc', please set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices'. [/home/abraham/src/OpenPaymentsService/HubDePagos.csproj]

Build FAILED.

Startup.cs(54,13): warning MVC1005: Using 'UseMvc' to configure MVC is not supported while using Endpoint Routing. To continue using 'UseMvc', please set 'MvcOptions.EnableEndpointRouting = false' inside 'ConfigureServices'. [/home/abraham/src/OpenPaymentsService/HubDePagos.csproj]
Installers/DbInstaller.cs(19,68): error CS0246: The type or namespace name 'ServerVersion' could not be found (are you missing a using directive or an assembly reference?) [/home/abraham/src/OpenPaymentsService/HubDePagos.csproj]
    1 Warning(s)
    1 Error(s)

Time Elapsed 00:00:01.67
```

### Further technical details

MySQL version: Mariadb 10.14.12
Operating system: Archlinux
Pomelo.EntityFrameworkCore.MySql version: 3.0.1
Microsoft.AspNetCore.App version: 3.1.2

### hypothesis

an statement like this is missing:

`using Pomelo.EntityFrameworkCore.MySql.Storage;`

at least that worked for me

--- a/README.md
+++ b/README.md
@@ -80,7 +80,6 @@ Add `Pomelo.EntityFrameworkCore.MySql` to the services configuration in your the
 using System;
 using Microsoft.EntityFrameworkCore;
 using Pomelo.EntityFrameworkCore.MySql.Infrastructure;
-using Pomelo.EntityFrameworkCore.MySql.Storage;
 
 // replace "YourNamespace" with the namespace of your application
 namespace YourNamespace
@@ -95,7 +94,7 @@ namespace YourNamespace
                 // replace with your connection string
                 .UseMySql("Server=localhost;Database=ef;User=root;Password=1234;", mySqlOptions => mySqlOptions
                     // replace with your Server Version and Type
-                    .ServerVersion(new ServerVersion(new Version(8, 0, 18), ServerType.MySql))
+                    .ServerVersion(new Version(8, 0, 18), ServerType.MySql)
             ));
         }
     }

--- a/README.md
+++ b/README.md
@@ -80,6 +80,7 @@ Add `Pomelo.EntityFrameworkCore.MySql` to the services configuration in your the
 using System;
 using Microsoft.EntityFrameworkCore;
 using Pomelo.EntityFrameworkCore.MySql.Infrastructure;
+using Pomelo.EntityFrameworkCore.MySql.Storage;
 
 // replace "YourNamespace" with the namespace of your application
 namespace YourNamespace

